loom/script/runtime/lsLuaState.cpp

```diff
@@ -86,8 +85,8 @@ void LSLuaState::open()
 {
     assert(!L);
 
-    L = lua_newstate(lsLuaAlloc, NULL);
-    //L = luaL_newstate();
+    //L = lua_newstate(lsLuaAlloc, NULL);
+    L = luaL_newstate();
```

Comment:
We didn't really investigate it a lot, but with this (and going to default allocator) it was easier to isolate crashes (they didn't corrupt random lua junk). So I can revert it back until we decide differently.